app/script/view_model/content/GroupCreationViewModel.js

```diff
@@ -87,6 +89,12 @@ z.ViewModel.content.GroupCreationViewModel = class GroupCreationViewModel {
       }
       return $(document).off('keydown.groupCreation');
     });
+    this.stateIsParticipants.subscribe(stateIsParticipants => {
+      if (stateIsParticipants) {
+        return window.setTimeout(() => this.showContacts(true));
```

Comment:
`setTimeout` -> `requestAnimationFrame`
Might be a little bit more expressive.

Reply:
I tried. Behavior is differently then and it does not address the problem. Will stick to this.